app/workers/transactional_email_analytics_job.rb

```diff
@@ -1,5 +1,13 @@
 # frozen_string_literal: true
 
+unless Rails.application.config.eager_load
+  # constantize all the mailers so TransactionalEmailMailer.descendants has something
+  # inspired by https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#single-table-inheritance
+  Dir['app/mailers/*.rb']
+    .collect { |mailer| %r{app/mailers/(.*)\.rb}.match(mailer)[1] }
+    .map { |x| x.camelize.constantize }
+end
```

Comment:
This really makes my 🕷️ sense tingle. 

Does this come out once the zeitwerk flag is flipped?

Reply:
It's sort of the opposite case, the zeitwerk loader needs this because TransactionalEmailAnalyticsJob uses `descendants` which doesn't autoload unnamed files.  So when it's loaded it needs to eager-load it's descendants.

Reply:
This is the method that uses the meta descendants
https://github.com/department-of-veterans-affairs/vets-api/blob/56ecc229ba6ec8f51e460dc50bfa12ce6eebcb8e/app/workers/transactional_email_analytics_job.rb#L45

Reply:
I guess alternatively we could explicitly list the TransactionalEmailAnalyticsJobs in a constant.  Just in my original zeitwerk PR I was trying to make minimal changes.  As an isolated PR it might make sense to change it up.

Reply:
I realized part of the root of the tingling here is that the mailers aren't really following Rails norms. Instead of inheriting, traditionally you'd have each different email as a method in the container (eg ApplicationMailer, TransactionalMailer) and then all of these mailer controllers start to hew to the convention.

The transactional analytics job i need to dig into more. It seems like there's no real need to dynamically ask "what emails need the analytics" when you can tell it "these jobs need analytics". Solves both problems but now we're getting way closer to rewriting code :/

Reply:
I've never written a mailer before, but maybe we could get that rewrite into some of this tech debt stuff?

Reply:
I'm a ➕ 1 on re-writing as part of tech debt 👍 I'm not familiar with the mailers either and this probably requires some digging and better understanding overall

Reply:
> we could explicitly list the TransactionalEmailAnalyticsJobs in a constant

I'm in favor of this ☝️ solution because it's simplicity. Perhaps, we could use this Dir/collect/map loop in a spec instead to verify that the new constant we make contains all of the "descendents" we expect.